chat-plugins/poll.js

```diff
@@ -219,12 +219,12 @@ exports.commands = {
 					room.poll.end();
 					delete room.poll;
 				}), (timeout * 60000));
-				room.add("The timeout for the poll was set to " + timeout + " minutes.");
-				return this.privateModCommand("(The poll timeout was set to " + timeout + " minutes by " + user.name + ".)");
+				room.add("The poll will automatically end in " + timeout + " minutes.");
```

Comment:
And correspondingly, "The poll timer was turned on: the poll will end in X minutes"